web/portal/client/src/entities/ExternalCallFilter/ExternalCallFilter.tsx

```diff
@@ -224,6 +231,20 @@ const externalCallFilter: EntityInterface = {
     ...defaultEntityBehavior.acl,
     iden: 'ExternalCallFilters',
   },
+  marshaller: (
+    values: MarshallerValues,
+    properties: PartialPropertyList,
+    whitelist: string[] = []
+  ): MarshallerValues => {
+    const response = DefaultMarshaller(values, properties, whitelist);
+
+    if (response.outOfScheduleEnabled && !response.outOfScheduleTargetType) {
+      // Force value in residential clients
+      response.outOfScheduleTargetType = 'number';
+    }
+
```

Comment:
Right now, outOfScheduleTargetType can be saved without value. Will this affect other clients aside from Residential?